src/components/PersonLink.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { Person } from '../types';
+import classNames from 'classnames';
+import { NavLink, useLocation } from 'react-router-dom';
+
+interface Props {
+  person: Person;
+}
+
+export const PersonLink: React.FC<Props> = ({ person }) => {
+  const { pathname } = useLocation();
+
+  const location = pathname.split('/');
+
+  return (
+    <tr
+      key={person.name}
```

Comment:
Using `person.name` as a key might not be unique if there are multiple people with the same name. Consider using a unique identifier, such as `person.slug` or another unique property, to ensure that each element has a unique key.